tests/callSitespeed.test.js

```diff
@@ -0,0 +1,7 @@
+const callSitespeed = require('../src/callSitespeed');
+
+test('the response is good', async () => {
```

Comment:
In the Ruby world, we have libraries for mocking out expensive external API calls. I believe this should be done here as well, otherwise this test suite will quickly become unmanageable.

Reply:
It's not a very pleasant test for sure. There is not much to test aside from the actual API call, though. Mocking it would basically involve setting errors = [], and then testing if errors.length = 0.

What I think would be a reasonable solution would be to move the test out of the actual function and into an expected long-running integration test. The integration test would still take 20-30 seconds.